### DISCUSSION TITLE
The converting Python library, to create d3plot files from OpenRadioss animation files, is publically released !

### DISCUSSION THREAD

**Discussion by MarianBulla** [OpenRadioss/OpenRadioss#2790] [Announcements]
Please review the message from Vortex-Radioss here:

https://github.com/orgs/OpenRadioss/discussions/2361

Thanks and best regards,
Marian

**Comment by Vortex-CAE** [comment]
Thank you Marian, I would like to thank all the members of the OpenRadioss community again for their help and assistance in trialling and testing the code. We had some really useful feedback, which helped speed-up the date of release significantly.

The library can be found here:

https://github.com/Vortex-CAE/Vortex-Radioss
